tools/api-generator/commands.go.tmpl

```diff
@@ -43,6 +43,7 @@ var Commands = GroupedAndSortedCommands{
 								IsArray:	{{ .Type.IsArray }},
 								Type:		`{{ .Type.Type }}`,
 							},
+							Aliases: []string{ {{ range $i, $alias := .Aliases }}{{if $i}},{{end}}`{{ $alias }}`{{end}} },
```

Comment:
can this be 
```suggestion
							{{if .Aliases }} Aliases: []string{ {{ range $i, $alias := .Aliases }}{{if $i}},{{end}}`{{ $alias }}`{{end}} }, {{end}}
```

Reply:
I was hesitating between what I have and what you suggested.
I've updated the code.

Reply:
I still se a couple of empty slices initialization which I was trying to avoid

Reply:
That's fixed now. I blindly accepted the suggestion, but I forgot that there's a second piece of code that does the same thing.